File: leetcode.com/1450_number_of_students_doing_homework_at_a_given_time/solution.c
int busyStudent(int* startTime, int startTimeSize, int* endTime, int endTimeSize, int queryTime)
{
    int count = 0;

    for (int i = 0 ; i < startTimeSize ; i++) {
        if (startTime[i] <= queryTime && queryTime <= endTime[i])
            count++;
    }

    return count;
}

File: leetcode.com/1450_number_of_students_doing_homework_at_a_given_time/solution.go
func busyStudent(startTime []int, endTime []int, queryTime int) int {
    count := 0

    for i := 0 ; i < len(startTime) ; i++ {
        if startTime[i] <= queryTime && queryTime <= endTime[i] {
            count += 1
        }
    }

    return count
}
